environment/MetaData.go

```diff
@@ -0,0 +1,96 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build integration
```

Comment:
One thing I wanted to ask and doesn't necessarily pertain specifically to this PR: Do we still need the integration build tag? This is a separate package now, so I feel like this is unnecessary. We don't have to make that change in this PR but wanted to ask.

Reply:
no, I was going to do that after test package separation and then got lazy. Will have to change in both public/private repo + in test repo. Thought I could punt that in favor of adding more tests & delivering alpha :D